docs/source/user_guide/installation/GNU-linux.rst

```diff
@@ -130,21 +130,6 @@ Currently, only GNU/Linux and macOS are supported.
 
    Assuming that SUNDIALS was installed as described :ref:`above<user-install-label>`.
 
-Optional - JaxSolver
---------------------
-
-Users can install ``jax`` and ``jaxlib`` to use the Jax solver.
-Currently, only GNU/Linux and macOS are supported.
-
-GNU/Linux and macOS
-~~~~~~~~~~~~~~~~~~~
-
-.. code:: bash
-
-	  pybamm_install_jax
-
-The ``pybamm_install_jax`` command is installed with PyBaMM. It automatically downloads and installs jax and jaxlib on your system.
-
```

Comment:
This should not be removed. The bash code should just change to `pip install "pybamm[jax]"`